app/views/dashboard/_activity.html.erb

```diff
@@ -65,11 +65,10 @@
         </label>
       </a>
       <!--
-      <li class="divider"></li>
-      <li class="dropdown-header"><%=translation('tag.index.sort_by')%></li>
-      <li><a href="#"><a href="#">Post date</a></li>
-      <li><a href="#"><a href="#">Recent activity</a></li>
-      -->
+<!--       <li class="divider"></li>
+      <li class="dropdown-header"><%=translation('tag.index.sort_by')%></li> -->
+   <li><a href="#"><a href="#"><%= translation('dashboard._activity.post_date') %></a></li>
+      U<li><a href="#"><a href="#"><%= translation('dashboard._activity.recent_activity') %></a></li>
     </div>
```

Comment:
@Axuby, these `<!--` and `-->` are HTML comments, and it looks like you moved their location. The `-->` should come after the last `</li>` here.

Also there is now a `U` character that needs to be deleted, and the indentation is slightly off.